app/models/project/download.rb

```diff
@@ -43,6 +43,7 @@ def rescue_reopen_error
   # @param file [String]
   def unzip_to_source_files(file)
     Zip::File.open_buffer(file) do |zip_file|
+      if zip_file.glob('**/.rubocop_todo.yml').present? then @project.has_todo = true end
```

Comment:
You can rewrite this as `@project.has_todo = zip_file.glob('**/.rubocop_todo.yml').present?`.